deploy/cloudwatch-agent.json

```diff
@@ -15,6 +15,11 @@
             "file_path": "/var/log/nginx/access.log",
             "log_group_name": "/tnlcf/APP_ENV/nginx/access",
             "log_stream_name": "{instance_id}"
+          },
+          {
+            "file_path": "/var/log/clamav/clamd.log",
+            "log_group_name": "/tnlcf/APP_ENV/clamav/clamd",
```

Comment:
This will need to be added to the DataDog Lambda function so these logs are pushed there.